geode-core/src/main/java/org/apache/geode/internal/cache/TXCommitMessage.java

```diff
@@ -104,7 +104,7 @@
   private transient boolean wasProcessed;
   private transient boolean isProcessing;
   private transient boolean dontProcess;
-  private transient boolean departureNoticed = false;
+  private transient volatile boolean departureNoticed = false;
```

Comment:
I'm going to remove this "volatile" because all access to the variable is under synchronization.  It's external synchronization for some reason, but it's in place so the volatile attribute isn't needed.